src/gate_detect/src/PoleDetection.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+ * Created By: Cameron Newton
+ * Created On: February 27th, 2018
+ * Description: Hold 2 lines that are determined to represent a pole
+ */
+
+#include <PoleDetection.h>
+
+PoleDetection::PoleDetection() {
+    _m = 0;
+
+    _b = 0;
+}
+
+PoleDetection::PoleDetection(cv::Vec4i side1,
+                             cv::Vec4i side2,
+                             float m,
+                             float b) {
+    _side1 = side1;
+
+    _side2 = side2;
+
+    _m = m;
+
+    _b = b;
+}
```

Comment:
See above comments on using initializer lists for constructors for both constructors here.